### DISCUSSION TITLE
Вычитание треков под Firefox

### DISCUSSION THREAD

**Discussion by KisoidPoh095** [Chimildic/YaMuTools#17] [Помощь с ошибкой] [answered]
У меня не заработало.
После нажатия кнопки крутится бесконечно "Загрузка плейлиста..." (плейлист маленький, так что дело не в объёмах)
Скрин с ошибкой в консоли: https://fastpic.org/view/115/2021/1001/_3c8769fa75a082a3bf84083933def733.png.html

Браузер: Firefox 81.0.2 (64-битный)
Сижу через проксю, если это важно. Но прокся полностью работоспособна.

**UPD**

Если открыть страничку https://music.yandex.ru/handlers/library.jsx?owner=kipo095&filter=playlists прямо в браузере, то она открывается, но там пустой json-ответ

**Comment by Chimildic** [comment]
Пока затрудняюсь ответить в чем причина. В `chrome` с тем же кодом работает правильно. Сходу различий не нашел.

**Comment by Chimildic** [comment, marked as answer]
Должно работать корректно на Firefox с версии 0.7.6

**Reply by KisoidPoh095** [reply]
Да, теперь работает. Вычитание по недавним в истории тоже заработало.
